package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "animate.css",
-  "version": "3.7.2",
+  "version": "4.0.0",
```

Comment:
Please remove this as npm does this automatically  when publishing the module.

Reply:
Really? so I should revert this one?

Reply:
Yes, please. It's all automagic ;D

Reply:
Alright will do